src/extension.ts

```diff
@@ -1097,6 +1098,24 @@ async function getLogsForPod(pod: PodSummary, follow?: boolean) {
     }
 }
 
+async function getEvents(follow: kuberesources.EventDisplayMode, explorerNode?: explorer.ResourceNode) {
```

Comment:
extension.ts is getting too big, so we should componentize as much as possible. move this to `src/components/kubectl/events.ts`

Reply:
I was to ask about that , event we should think of moving lots of logical components out of extension.ts like `kuberrnetsCommands.ts`, `helmCommands.ts` etc., at first rather even now I find it hard to read the extension.ts :)